modules/core/04-channel/types/acknowledgement.go

```diff
@@ -70,5 +69,9 @@ func (ack Acknowledgement) Success() bool {
 // Acknowledgement implements the Acknowledgement interface. It returns the
 // acknowledgement serialised using JSON.
 func (ack Acknowledgement) Acknowledgement() []byte {
-	return sdk.MustSortJSON(SubModuleCdc.MustMarshalJSON(&ack))
+	res, err := json.Marshal(SubModuleCdc.MustMarshalJSON(&ack))
```

Comment:
```suggestion
	res, err := json.Marshal(&ack)
```
I think we switch to just marshaling the acknowledgement?

Reply:
Yes. I will just remove the `MustSortJSON` and leave the `MustMarshalJSON` untouch